core/src/main/java/com/cloud/storage/JavaStorageLayer.java

```diff
@@ -178,18 +183,25 @@ public long getSize(String path) {
     }
 
     @Override
-    public File createUniqDir() {
+    public File createUniqDir() throws IOException {
         String dirName = System.getProperty("java.io.tmpdir");
         if (dirName != null) {
             File dir = new File(dirName);
             if (dir.exists()) {
+                if (isWorldReadable(dir)) {
+                    if (dir.getAbsolutePath().equals("/tmp")) {
+                        s_logger.warn("The temp dir is /tmp");
```

Comment:
and than here of course
```suggestion
                        s_logger.warn(String.format("The temp dir is %s", STDTMP);
```